tools/linux/blk/blk_init.mk

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2024, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+ifeq ($(strip $(LIBVMM)),)
+$(error LIBVMM must be specified)
+endif
+ifeq ($(strip $(MICROKIT_BOARD)),)
+$(error MICROKIT_BOARD must be specified)
+endif
+
+blk_client_init: $(LIBVMM)/tools/linux/blk/blk_client_init
```

Comment:
Might be better to put all the files you want to dump into the root image into a subdir, one for each root image.
Maybe note and do that later

Reply:
I'm not sure how we'd do this in `blk_init.mk`. Scripts like `blk_client_init` would need to be copied into more than one rootfs. `blk_init.mk` does not know the layout of our system.

Reply:
Offline discussion: going to do this later, maybe as a part of the Lions OS Makefile generation.